### PR TITLE
Use newer data for stereo examples

### PR DESCRIPTION
**Pull request by dstansby**
Should fix the doc build

### CODE REVIEW

File: examples/units_and_coordinates/AIA_limb_STEREO.py [hunk 1]
```diff
@@ -22,13 +22,13 @@
 # early 2011 when the STEREO spacecraft were roughly 90 deg separated from the
```
Comment (nabobalis) [inline examples/units_and_coordinates/AIA_limb_STEREO.py:22]: Will need to update the example text as well to match.

**Review by nabobalis — APPROVED** [review]

**Comment by dstansby** [comment]
https://sunpy--5529.org.readthedocs.build/en/5529/generated/gallery/index.html#coordinates-times-and-units if anyone wants to check the examples are still fine - I've taken a look and they seem to be to me.

**Review by ayshih — CHANGES_REQUESTED** [review]
Stuff that ought to be changed in this PR:

- Plotting this new AIA image would greatly benefit from explicitly setting `vmin`.

Stuff that would be nice to also get in, but can be a separate PR:

- `AIA_limb_STEREO.py`
  - The ~dozen lines near at the end to annotate the HPC grid as seen by SDO on the STEREO image don't do anything because that HPC grid doesn't actually intersect with the edges of the plot area, so no ticks, tick labels, or axis labels are shown.  One way to "fix" this is to zoom in so that the plot is no longer full-disk.
- `SDO_to_STEREO_Coordinate_Conversion.py`
  - This example needs some re-thinking.  It's really about submapping the same region on two maps from different observers, and the explicit coordinate conversions are all superfluous (now).
  - I – of course – heartily recommend using `draw_quadrangle()` for the plotting of boxes:
Figure_4
Figure_6

**Comment by nabobalis** [comment]
> Oh rats, merged while I was typing this. Okay, I'll start a new PR

I opened https://github.com/sunpy/sunpy/issues/5533 to keep track of your suggestions.

**Pull request merged**